.github/workflows/dockerhub.yml

```diff
@@ -1,26 +1,15 @@
-name: DockerHub
+name: Docker Hub build
 
 on:
   push:
     branches:
-      - "carma-develop"
+      - "develop"
+      - "master"
+      - "release/*"
```

Comment:
I am not sure what these lines do? Is it saying to build these branches, or ignore them? carma-develop, master, release branches we do want to build right?

Reply:
I think I got my answer in private chat from Sai:
`ci.yml: it will build and run units for sonar scanner analysis but does not push any images to docker hub
dockerhub.yml: it will build and push of docker images when push occurs to develop, release branches, master but doesn't run any unit test or sonar scanner
docker.yml: it will only build for feature branches and Pullrequest is opened, changed , merged (edited) `